src/ayab/encoders.cpp

```diff
@@ -133,8 +133,8 @@ void Encoders::encA_rising() {
   m_direction = digitalRead(ENC_PIN_B) != 0 ? Direction_t::Right : Direction_t::Left;
 
   // Update carriage position
-  if ((Direction_t::Right == m_direction) && (m_position < END_RIGHT[static_cast<uint8_t>(m_machineType)])) {
-    m_position = m_position + 1;
+  if ((Direction_t::Right == m_direction)) {
```

Comment:
Yes, removing this test on machine limits is a good call. Blindly ignoring that the machine did move out of the expected bounds wasn’t going  to be help 😬 
In the future, maybe reporting the unexpected situation somehow (going into an error state?) could help catch unwarranted assumptions earlier.